module-apps/application-settings/windows/display-keypad/QuotesMainWindow.cpp

```diff
@@ -47,6 +47,18 @@ namespace gui
                                  quotesModel,
                                  gui::listview::ScrollBarType::Fixed);
 
+        list->emptyListCallback = [this]() {
+            navBar->setActive(nav_bar::Side::Left, false);
+            navBar->setActive(nav_bar::Side::Center, false);
+            application->refreshWindow(gui::RefreshModes::GUI_REFRESH_DEEP);
+        };
+
+        list->notEmptyListCallback = [this]() {
+            navBar->setActive(nav_bar::Side::Left, true);
+            navBar->setActive(nav_bar::Side::Center, true);
+            application->refreshWindow(gui::RefreshModes::GUI_REFRESH_DEEP);
```

Comment:
These callbacks do not return that list should be refreshed, therefore we have to call it ourselves? :)

Reply:
Yes it doesn't. Tried to remove it, however it would probably need to be modified in ListView and multiple changes out of scope of these fixes would need to follow.